aries-site/src/examples/components/anchor/AnchorInlineExample.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Anchor, Box, Paragraph } from 'grommet';
+
+export const AnchorInlineExample = () => {
+  return (
+    <Box align="center">
+      <Paragraph>
+        Embracing the courage and commitment to learn, share and serve with
+        uncompromising integrity, the HPE Design System will advance the way
+        people live and work. If you haven't yet, you can read about{' '}
+        <Anchor
+          label="Our Philosophy"
+          href="https://design-system.hpe.design/foundation/philosophy"
+        />{' '}
+        here.
```

Comment:
In the guidance above, we talk about the word "here" as being unhelpful, then end up using it as an example. Not an apples to oranges comparison, but could avoid anyone sensing a contradiction by removing. I think it reads just as well.
```suggestion
        />
        .
```